src/contracts/test/GPv2EncodingTestInterface.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "../libraries/GPv2Encoding.sol";
+
+contract GPv2EncodingTestInterface {
+    bytes32 public constant DOMAIN_SEPARATOR =
+        0x000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f;
+
+    function decodeSignedOrdersTest(
```

Comment:
I am not sure I understand how this is a test of `decodeSignedOrders`. Looks more like some sort of gas estimation method...